src/main/java/com/es/phoneshop/utils/ParseValidator.java

```diff
@@ -3,10 +3,14 @@
 public class ParseValidator {
 
     public static boolean validateQuantity(String quantityStr) {
+        quantityStr = removeMinusesFromString(quantityStr);
         if (quantityStr.matches("^\\d+([\\.\\,]\\d+)?$")) {
             return true;
         } else {
             return false;
         }
     }
+    public static String removeMinusesFromString(String input) {
```

Comment:
validator is not changing the content which it is validating
it says if the input is valid or not